src/main/java/net/imagej/ops/threshold/ThresholdNamespace.java

```diff
@@ -122,6 +126,24 @@
 		return result;
 	}
 
+	// -- binary mask --
+
+	@OpMethod(op = net.imagej.ops.threshold.binaryMask.DefaultBinaryMask.class)
```

Comment:
I'd not merge this, because it introduces redundant concepts to existing functionality.